README.md

```diff
@@ -29,7 +29,7 @@ Gradle.
 ### Start Authoring Your AI
 
 The directory you copied already has a class
-named [YourCustomAI](https://github.com/followsclosely/connect-four/blob/master/entries/copy-me-to-get-started/src/main/java/YourCustomAI.java)
```

Comment:
This file should not be changed.

Reply:
No clue how that happened, I can't even figure out how to make edits to that file.